src/components/PersonList.tsx

```diff
@@ -0,0 +1,41 @@
+import classNames from 'classnames';
+import { Person } from '../types/Person';
+
+type Props = {
+  visiblePeople: Person[],
+  onSelected?: (person: Person) => void,
+};
+
+export const PersonList: React.FC<Props> = ({
+  visiblePeople,
+  onSelected = () => {},
+}) => {
+  return (
+    <div className="dropdown-menu" role="menu">
+      <div className="dropdown-content">
+        { visiblePeople.length !== 0
+          ? (visiblePeople.map(people => (
+            <div className="dropdown-item" key={people.slug}>
+              <a
+                href="//"
+                onClick={(event) => {
+                  event.preventDefault();
+                  onSelected(people);
+                }}
```

Comment:
better to create separate function
```suggestion
                onClick={onPersonClick}
```